pages/dkp/kommander/2.1/backup-and-restore/index.md

```diff
@@ -197,6 +197,46 @@ To restore cluster data on demand from a selected backup snapshot available in t
 velero restore create --from-backup BACKUP-NAME
 ```
 
+#### Restoring Kommander Management Cluster
+
+<p class="message--note"><strong>NOTE: </strong>When restoring a backup to the Kommander Management Cluster, you must adjust configuration to avoid restore errors.
+</p>
+
+Before running a restore make sure that the following `ResourceQuota` configuration is not configured on your cluster:
+
+```sh
```

Comment:
We are using bash for input and sh for output, so the copy icon only appears in executable commands and customers don't try to execute output :) 
```suggestion
```bash
```